### PR TITLE
fix(configuration): CHK-1688 modified jwt validity

### PR DESCRIPTION
**Pull request by Fernando-Granato**
#### List of Changes
Removed JWT_TOKEN_VALIDITY_TIME_MILLIS configuration. To configure the validity of the jwt we will use the same value as the paymentToken duration configuration variable

#### Motivation and Context
he change was made to align the duration of the eCommerce session with that of the pm 

#### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/main/java/it/pagopa/transactions/utils/JwtTokenUtils.java [hunk 1]
```diff
@@ -23,26 +24,26 @@ public class JwtTokenUtils {
 
     public static final String TRANSACTION_ID_CLAIM = "transactionId";
 
-    private final int tokenValidityTimeMillis;
+    private final int tokenValidityTimeSecond;
```
Comment (giovanniberti) [inline src/main/java/it/pagopa/transactions/utils/JwtTokenUtils.java:27]:
```suggestion
    private final int tokenValidityTimeSeconds;
```

File: src/main/java/it/pagopa/transactions/utils/JwtTokenUtils.java [hunk 2]
```diff
@@ -23,26 +24,26 @@ public class JwtTokenUtils {
...
-        this.tokenValidityTimeMillis = tokenValidityMillis;
+        this.tokenValidityTimeSecond = tokenValiditySecond;
     }
 
     public Mono<String> generateToken(TransactionId transactionId) {
         try {
             Calendar calendar = Calendar.getInstance();
             Date issuedAtDate = calendar.getTime();
-            calendar.add(Calendar.MILLISECOND, tokenValidityTimeMillis);
+            calendar.add(Calendar.MILLISECOND, (int) Duration.ofSeconds(tokenValidityTimeSecond).toMillis());
```
Comment (giovanniberti) [inline src/main/java/it/pagopa/transactions/utils/JwtTokenUtils.java:41]: Is it different from
```suggestion
            calendar.add(Calendar.SECOND, tokenValidityTimeSecond);
```

Comment (Fernando-Granato) [reply src/main/java/it/pagopa/transactions/utils/JwtTokenUtils.java:41]: OK you’re right

Comment (Fernando-Granato) [reply src/main/java/it/pagopa/transactions/utils/JwtTokenUtils.java]: Fixed with 23d0ce22fb07b52d820ce166e176eb199f18a92c

File: src/main/java/it/pagopa/transactions/utils/JwtTokenUtils.java (hunk 1 above)
Comment (Fernando-Granato) [reply src/main/java/it/pagopa/transactions/utils/JwtTokenUtils.java]: Fixed with 23d0ce22fb07b52d820ce166e176eb199f18a92c

File: src/test/java/it/pagopa/transactions/utils/JwtTokenUtilsTests.java [hunk 3]
```diff
@@ -17,9 +18,9 @@
 class JwtTokenUtilsTests {
     private static final String STRONG_KEY = "ODMzNUZBNTZENDg3NTYyREUyNDhGNDdCRUZDNzI3NDMzMzQwNTFEREZGQ0MyQzA5Mjc1RjY2NTQ1NDk5MDMxNzU5NDc0NUVFMTdDMDhGNzk4Q0Q3RENFMEJBODE1NURDREExNEY2Mzk4QzFEMTU0NTExNjUyMEExMzMwMTdDMDk";
 
-    private static final int TOKEN_VALIDITY_TIME = 10000;
+    private static final int TOKEN_VALIDITY_TIME_SECOND = 900;
```
Comment (giovanniberti) [inline src/test/java/it/pagopa/transactions/utils/JwtTokenUtilsTests.java:21]:
```suggestion
    private static final int TOKEN_VALIDITY_TIME_SECONDS = 900;
```

**Review by giovanniberti — APPROVED** [review]

**Pull request merged**